app/register/page.tsx

```diff
@@ -56,15 +56,14 @@ const Register = (): JSX.Element => {
     if (isSignedIn) {
       router.push('/weeklyPicks');
     }
-  }, [isSignedIn]);
+  }, [isSignedIn, router]);
```

Comment:
Why is router a dependency? This will rehydrate when a route changes I believe.

Reply:
I added this in as per the warning from the eslint rules you added.

![CleanShot 2024-06-18Code001219@2x](https://github.com/LetsGetTechnical/gridiron-survivor/assets/40150036/b715c1b0-18a4-4d0a-b328-933e7d76ad7c)

Reply:
Updated in latest commit.

Reply:
Yellow = warning. :D